src/examples/rest/client.cpp

```diff
@@ -37,10 +37,17 @@
 //
 
 
-#include <httplib.h>
 #include <iostream>
 #include <cstring>
 
+// save diagnostic state
+#pragma GCC diagnostic push
+// turn off the specific warning. Can also use "-Wall"
+#pragma GCC diagnostic ignored "-Wsign-compare"
+#include <httplib.h>
```

Comment:
the `SYSTEM` include did not fix the warnings generated in httplib (I wonder why?), so I had to hack with `#pragma` here and in `server_core.hpp`.

Reply:
> the SYSTEM include did not fix the warnings

I don't know of a way to get around the warnings other than with the #pragma.